apps/base-docs/docs/account-abstraction-with-base-paymaster/intro-to-account-abstraction.md

```diff
@@ -0,0 +1,163 @@
+
+To summarize, Account Abstraction enables smart contract accounts to initiate user operations, similar to how an EOA would initiate a transaction. However, unlike EOAs, smart contract accounts are programmable and can enable a number of incredible features, such as:
+
+- **Sponsored transactions**: Allow application owners to cover the users gas fees with a paymaster or allow a user to use something other than ETH (USDC, for example) to cover gas.
+- **Arbitrary verification logic**: Verify transactions with custom logic.
+- **Account recovery**: Create account recovery features for when user lose private keys.
+- **Batching transactions**: Change the user experience so that multiple transactions can be submitted at once.
+
+And more!
+
+Let's dive into how Account Abstraction enables this.
+
+---
+
+## Overview of ERC-4337 - Account Abstraction
+
+"Account Abstraction" comes from [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337). The proposal itself is a dense read, but one of stated goals of the proposal is to,
+
+> allow users to use smart contract wallets containing arbitrary verification logic instead of EOAs as their primary account. Completely remove any need at all for users to also have EOAs (as status quo SC wallets and EIP-3074 both require)
+
+In other words, the propsal seeks to allow users to use "smart contract wallets" _instead_ of EOAs to transact onchain.
+
+By the way, you'll often see the terms "smart contract wallet", "smart contract account" or just "smart account" used interchangeably.
+
+At first glance, you may be asking yourself if using smart contract accounts solve our problem. What's the difference between a smart contract account (or contract account) and an "Externally Owner Account"?
+
+According to [https://ethereum.org/en/developers/docs/accounts/#key-differences](https://ethereum.org/en/developers/docs/accounts/#key-differences), the differences are:
+
+**Externally Owned Account**
+
+- Creating an account costs nothing
+- Can initiate transactions
+- Transactions between externally-owned accounts can only be ETH/token transfers
+- Made up of a cryptographic pair of keys: public and private keys that control account activities
+
+**Contract Accounts**
+
+- Creating a contract has a cost because you're using network storage
+- Can only send transactions in response to receiving a transaction
+- Transactions from an external account to a contract account can trigger code which can execute many different actions, such as transferring tokens or even creating a new contract
+- Contract accounts don't have private keys. Instead, they are controlled by the logic of the smart contract code
+
+As we stated before, smart contract wallets _would_ solve our problems, **but**, according to the information above, they can't initiate transactions. And since they cannot initiate transactions, we still need EOAs and those EOAs still need to pay for gas with ETH.
```

Comment:
"according to the information above" is unnecessary.  You can just state the fact.